libs/components/lookup/src/lib/modules/autocomplete/autocomplete.component.ts

```diff
@@ -300,6 +303,20 @@ export class SkyAutocompleteComponent implements OnDestroy, AfterViewInit {
   @Input()
   public searchAsyncDisabled: boolean | undefined = false;
 
+  /**
+   * Allows the user to specify arbitrary values not in the search results.
+   * @default false
+   */
+  public allowAnyValue = input(false, { transform: booleanAttribute });
+
+  /**
+   * Highlights the search text in each search result. Set this to false
+   * when your search finds results that are not exact text matches, e.g.
+   * returning "Bob" for the term "Robert."
+   * @default false
```

Comment:
The default listed in the JSDoc is different from the actual default. Intentional?

Reply:
Good catch! Fixed.